backend/src/core/utils/attachment.ts

```diff
@@ -9,4 +18,7 @@ const ensureAttachmentsFieldIsArray = (
   return attachments
 }
 
-export { ensureAttachmentsFieldIsArray }
+export const getAttachmentHash = (content: Buffer): string => {
```

Comment:
Think you mentioned this is unnecessary and can be refactored. Can put in a TODO here?

Reply:
Lemme just include it in this PR